scripts/dict_audit_symbols.py

```
@@ -0,0 +1,299 @@
+        stem = dict_path.name
+        for ext in (".dict", ".idx", ".syn"):
+            f = parent / f"{stem}{ext}"
+            if f.exists():
+                files_to_scan.append(f)
+
+    if not files_to_scan:
+        print(f"Error: No .dict, .idx, or .syn files found for path {dict_path}", file=sys.stderr)
+        sys.exit(1)
+
+    # 1. Parse dictionary content and count frequencies
+    print(f"Scanning files:")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)